src/api/mod.rs

```diff
@@ -1,9 +1,25 @@
 mod client;
 mod server;
 
+const LISTEN_ADDR: &str = "127.0.0.1:";
+const URI_PREFIX: &str = "http://";
+
 pub use client::LocalClient;
 pub use helium_proto::services::local::{
     ConfigReq, ConfigRes, ConfigValue, EcdhReq, EcdhRes, HeightReq, HeightRes, PubkeyReq,
     PubkeyRes, SignReq, SignRes,
 };
 pub use server::LocalServer;
+
+pub fn listen_addr(port: u16) -> String {
+    let mut address = LISTEN_ADDR.to_string();
```

Comment:
use the format! macro

Reply:
like `format!("{}:{}", LISTEN_ADDR, port)` and define LISTEN_ADDR as just 127.0.0.1